spring-batch-docs/asciidoc/readersAndWriters.adoc

```diff
@@ -1796,32 +1798,32 @@ orientated mechanism for manipulating a cursor. A
 `ResultSet` maintains a cursor to the current row
 of data. Calling `next` on a
 `ResultSet` moves this cursor to the next row.
-Spring Batch cursor based `ItemReaders` open the a cursor on
-initialization, and move the cursor forward one row for every call to
+Spring Batch cursor-based `ItemReader` implementation open a cursor on
```

Comment:
If you make it plural, you have to make "cursor" plural, too. Instead, I would do this:

The Spring Batch cursor-based `ItemReader` implementation opens....

(The differences are the "A" at the beginning and the "s" on "open".

Otherwise, it sounds like there are multiple cursor-based implementations, and the reader's logical questions will be: "What are they?" "How do I know which one to use?" And so on.

Reply:
Why would we need to make "cursor" plural?  Spring Batch has multiple cursor based `ItemReader` implementations (`JdbcCursorItemReader`, `HibernateCursorItemReader`).  Each one opens up a single cursor.  This section is talking about the general idea of cursors and implementing readers based upon them.

Reply:
That's my point. You don't want to make "cursor" plural, because each ItemReader opens only one cursor.

That's why I recommend writing:

The Spring Batch cursor-based `ItemReader` implementation opens a cursor....

Reply:
But doesn't your recommendation imply there is only one implementation?